week-2/implement/bmi.js

```diff
@@ -13,3 +13,16 @@
 // Given someone's weight in kg and height in metres
 // When we call this function with the weight and height
 // Then it returns their Body Mass Index to 1 decimal place
+
+
+
+function heightSquare(h){
+    return h*h;
```

Comment:
We can also put the whole equation within the function as well which I have realised today, that's also an alternate way